components/combobox/css/index.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021-2022 Gemeente Utrecht
+ * Copyright (c) 2021-2022 Frameless B.V.
+ */
+
+@import "./mixin";
+
+.utrecht-combobox {
+  @include utrecht-combobox;
+}
+
+.utrecht-combobox_list {
+  @include utrecht-combobox_list;
+}
+
+.utrecht-combobox_list-item {
+  @include utrecht-combobox_list-item;
+}
+
+.utrecht-combobox_list-item:hover {
+  @include utrecht-combobox_list-item--hover;
+}
+
+.utrecht-combobox_list-item[aria-selected="true"] {
+  @include utrecht-combobox_list-item--highlighted;
+}
+
+.utrecht-combobox_list-item--selected {
+  @include utrecht-combobox_list-item--selected;
+}
+
+.utrecht-combobox__input-wrapper {
+  @include utrecht-combobox__input-wrapper;
+}
+
+.utrecht-combobox__button {
+  @include utrecht-combobox__button;
+}
+
+.utrecht-combobox__button[aria-expanded="true"] {
+  &.utrecht-combobox-icon {
```

Comment:
Moet deze `&` hier wel staan?

Reply:
Ik heb het opgelost door een ander icon te gebruiken in plaats van te roteren